cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -150,6 +150,10 @@ type ControlPlaneComponent struct {
 
 	// ExtraVolumes is an extra set of host volumes, mounted to the control plane component.
 	ExtraVolumes []HostPathMount
+
+	// ExtraEnvs is an extra set of envs to pass to the control plane component, e.g. GOMAXPROCS.
```

Comment:
are we ok with the name extraenvs? @SataQiu @pacoxu

Reply:
seems ok